core/templates/domain/exploration/ExplorationObjectFactory.ts

```diff
@@ -242,6 +245,7 @@ export class ExplorationObjectFactory {
       explorationBackendDict.title,
       explorationBackendDict.next_content_id_index,
       explorationBackendDict.language_code,
+      explorationBackendDict.exploration_metadata,
```

Comment:
This is wrong. The exploration dict in the backend does not have a field called exploration_metadata.